### DISCUSSION TITLE
No menu When clicking SSH

### DISCUSSION THREAD

**Discussion by AlJeske** [NightscoutFoundation/xDrip#2731] [General]
This is exactly what happened to me, no menu (connecting to serial ports disabled) I have already been using Xdrip+ and want to be able to show my Endocrinologist the data. Setup is: Dexcom G6=>Xdrip+=>Aaps. If there is a better way for the Endo to see my progress, let me know and I will forego the Nightscout-google cloud endeavor. No Menu When I click SSH.
Sorry I didn't know how to start the discussion. I hope this is the right place.

**Comment by Navid200** [comment]
When you say to show data to your Endocrinologist, do you mean to create a report to show him, or  to let him see your readings all the time as a follower?

If you want someone to follow you, there is an alternative.  That's xDrip Sync, which is much simpler than Nightscout to set up.
You can see here how to set it up:
https://youtu.be/LcgjfbYcWkE

Regardless, Nightscout has other benefits.  it can serve as a repository of your readings.  Therefore, if one day you lose your phone, you will not lose all your readings.  They will all be on Nightscout.

Now, if you like to troubleshoot your Nightscout:
Please click on SSH and wait for the windows to come up.  Please take a screenshot and post here so that I can see what you see.

**Reply by AlJeske** [reply]
![image](https://user-images.githubusercontent.com/50727751/224875303-ffcafcee-1c08-416e-b2e3-0e7e337a326d.png)
Nothing happens when I click SSH. The idea was to create a report for him.

**Reply by Navid200** [reply]
You say nothing happens when you click on SSH.
I don't even see an instance there.

Did you follow this guide?
https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

**Reply by AlJeske** [reply]
What can I show you if nothing happens? And yes that is the guide I'm following. I waited at least 15 min. I showed you the page where I clicked on the instance-1 page and showed that connecting to a serial port is disabled--wondered if that has anything to do with problem.

**Reply by Navid200** [reply]
It looks like you have clicked on the instance instead of clicking on SSH on the right side.

Click on the arrow I have circled in the attached image to exit the instance details.
![224875303-ffcafcee-1c08-416e-b2e3-0e7e337a326d](https://user-images.githubusercontent.com/51497406/224884497-d5e09c05-c282-47ad-b705-c199f23a4d3f.png)

This page explains how you can use SSH to connect to the machine.
https://navid200.github.io/xDrip/docs/Nightscout/Terminal.html

**Reply by AlJeske** [reply]
![image](https://user-images.githubusercontent.com/50727751/224886981-422ba71e-c62c-4abd-bb5f-d7494a24903f.png)

**Comment by AlJeske** [comment]
Thanks, I'll check it out.

**Comment by AlJeske** [comment]
Not true. I clicked on SSH many times before I clicked on instance 1

**Reply by Navid200** [reply]
Post a screenshot of what you clicked on.
It should not be on the instance detail page that you are showing.  
It should be here:
![SSH](https://user-images.githubusercontent.com/51497406/224885997-533513a1-b574-47e0-94d0-aeff50fe77ce.png)

**Reply by AlJeske** [reply]
![image](https://user-images.githubusercontent.com/50727751/224887115-25feb3c5-41fa-4dce-89f1-7d0fb935f43d.png)

**Reply by Navid200** [reply]
When did you create this Nightscout, a few months ago or a few days ago?

**Comment by AlJeske** [comment]
Yesterday.

**Reply by Navid200** [reply]
When you created the machine yesterday, were you able to use SSH to connect?

**Reply by AlJeske** [reply]
no

**Comment by AlJeske** [comment]
Wait, if you mean nigthscout itself many months ago

**Reply by Navid200** [reply]
Sorry, I have a hard time understanding you.

If you created this machine a long time ago, it is possible you are experiencing this problem:
https://navid200.github.io/xDrip/docs/Nightscout/FullDisk.html

**Reply by AlJeske** [reply]
Machine: yesterday, Nightscout originally in Heroku long time ago but never really used. So probably not fullDisk problem.

**Reply by Navid200** [reply]
If you created the machine yesterday, you can delete it and create a new one.

Follow the steps.  As soon as you get to a point that something unexpected happens, let me know.  You can look at all the steps in video clips.
If you experience anything different than what is shown in the video clips, please stop and let me know.  Please don't continue.

**Reply by AlJeske** [reply]
Ok will do.

**Reply by Navid200** [reply]
Thanks

After I see what happens, it is quite possible that I will see something that I have not explained clearly in the guide and I will then be able to improve the guide so that others will not experience the same problem.

Thanks

**Comment by AlJeske** [comment]
Ok I will do my best to let you know what went wrong with my attempt. I'm
using Firefox browser.

**Comment by AlJeske** [comment]
![image](https://user-images.githubusercontent.com/50727751/225026577-cf029759-13da-4de6-9725-c2a81f80ce85.png)
Navid , Overnight I had the idea to try a different browser with the VM I created. This is the screen shot of what happened when I copied the URL from Firefox to Edge. I then clicked on SSH and up came the: SSH-in-browser shown in the screeen shot. So looks like Google Cloud doesn't like Firefox browser???

**Reply by Navid200** [reply]
If the browser blocks pop-ups, SSH will not be able to connect because it needs to open a new window.

Thanks for reminding me of this.  I will add this note to the guide.

Please continue with the installation when you like and keep me posted.

Thanks

**Comment by AlJeske** [comment]
Will do, thanks!

**Comment by AlJeske** [comment]
![image](https://user-images.githubusercontent.com/50727751/225026577-cf029759-13da-4de6-9725-c2a81f80ce85.png)
Navid , Overnight I had the idea to try a different browser with the VM I created. This is the screen shot of what happened when I copied the URL from Firefox to Edge. I then clicked on SSH and up came the: SSH-in-browser shown in the screeen shot. So looks like Google Cloud doesn't like Firefox browser???

**Comment by AlJeske** [comment]
![image](https://user-images.githubusercontent.com/50727751/225200549-058b1f5c-f0fa-49d9-9e82-85d9aac94bf9.png)
Here is where I get stuck now. It recycles me to SSH again. Not sure about the VM User ID (couldn't find it on top right of VM screen) and Password

**Reply by Navid200** [reply]
I thought you said Edge worked and Firefox didn't.
Did you change Firefox permissions so that it would work?

**Reply by AlJeske** [reply]
Yes I did. Maybe I have to complete with Edge??? I will try with Edge tomorrow.

**Reply by Navid200** [reply]
So, do you experience the same problem with Edge?

**Comment by AlJeske** [comment]
Before I went to Edge I went to Firefox and allowed all popups whereas
previously allowed popups from only VM Console.cloud.google. It worked now
and I'm part 5: Setting up Nightscout. I appreciate your help with the
pop-up problem. I'm hoping to get through this 😔 without anymore problems.

**Comment by AlJeske** [comment]
👍 Setting up Nightscout now. Do I need to go to discussion to say RESOLVED?

**Reply by Navid200** [reply]
The discussions are quite flexible.
You can do whatever you like.  
I'm glad it worked out.

**Comment by AlJeske** [comment]
I APPRECIATE your help and your involvement with Xdrip+ and google
cloud/Nightscout!
